crypto/math/datatypes.c

```diff
@@ -302,19 +277,14 @@ void v128_left_shift(v128_t *x, int shift)
 
 int bitvector_get_bit(const bitvector_t *v, int bit_index)
 {
-    return _bitvector_get_bit(v, bit_index);
+    return /bitvector_get_bit(v, bit_index);
```

Comment:
??? Does this compile?

Reply:
It does, but only because it's within `#ifndef DATATYPES_USE_MACROS`, and that doesn't appear to be the case for either my local build (macOS, CMake, default config) or any of the CI builds.  Maybe we should remove that option?

Reply:
`DATATYPES_USE_MACROS` is defined unconditionally in `datatypes.h`, so yeah, removing its `#ifndef` conditions is probably a good idea.